src/contour/CMakeLists.txt

```diff
@@ -202,8 +202,11 @@ if(WIN32)
     set(CPACK_PACKAGE_ICON "${CMAKE_CURRENT_SOURCE_DIR}/res/images/contour-logo.ico")
     set(CPACK_WIX_UPGRADE_GUID "0E736497-2B72-4117-95E9-54EC6DC2432A")
 
-    include(DeployQt)
-    windeployqt(contour)
+    if(CONTOUR_BUILD_WITH_QT6)
```

Comment:
I think we still need that for Qt6 builds, but inside `DeployQt.cmake`, the qt version is still hardcoded to Qt 5. That should somehow make use of `CONTOUR_BUILD_WITH_QT6` to mandate what to use. 

Maybe we should even move away from `CONTOUR_BUILD_WITH_QT6` and introduce something like `USE_QTVERSION_MAJOR` cmake-var that can be either `5` or `6` for easier use of it. 🤔

Reply:
Indeed, win deploy qt for qt6 does not exist, what we need is to manually execute windeployqt.exe
I will check what we need to  do here

Reply:
<img width="938" alt="image" src="https://user-images.githubusercontent.com/56763/232740327-02a87aca-e39b-4b1b-a926-1c47cd0aa960.png">
I think what I did on windows, is simply (but sadly manually) changing this value from 5 to 6. That should not be done automatically of course. Am I being mislead? 🤔

Reply:
> <img alt="image" width="938" src="https://user-images.githubusercontent.com/56763/232740327-02a87aca-e39b-4b1b-a926-1c47cd0aa960.png">
> I think what I did on windows, is simply (but sadly manually) changing this value from 5 to 6. That should not be done automatically of course. Am I being mislead? thinking

oh, okay, i think we can easilty adjust cmake file to use inside contour, and then `USE_QTVERSION_MAJOR` can be introduced